src/core/webview/webviewMessageHandler.ts

```diff
@@ -632,6 +632,8 @@ export const webviewMessageHandler = async (
 				if (!exists) {
 					await safeWriteJson(mcpPath, { mcpServers: {} })
 				}
+
+				openFile(mcpPath) // kilocode_change
```

Comment:
```suggestion
				openFile(mcpPath)
```

This is also here in Roo: https://github.com/RooCodeInc/Roo-Code/blob/7ae74404f4197681e1847d368ecf07b365116e5e/src/core/webview/webviewMessageHandler.ts#L618